### PR TITLE
Rename app-arch/upx-ucl to app-arch/upx

### PR DESCRIPTION
**Pull request by winterheart**
Renaming package to more common name. Fixed all (R)DEPENDS in tree.
This PR is part of https://github.com/gentoo/gentoo/pull/12087

Signed-off-by: Azamat H. Hackimov <azamat.hackimov@gmail.com>

### CODE REVIEW

**Comment by winterheart** [comment]
@juippis , this is first part of upx package updating.

File: profiles/updates/3Q-2019 [hunk 1]
```diff
@@ -2,3 +2,5 @@ move dev-python/CoverageTestRunner dev-python/coverage-test-runner
 move media-plugins/kipi-plugins kde-apps/kipi-plugins
 move kde-misc/yakuake kde-apps/yakuake
 move kde-apps/kjots app-text/kjots
+move app-arch/upx-ucl app-arch/upx
+
```
Comment (mgorny) [inline profiles/updates/3Q-2019]: You've got invalid empty line here, and it crashed CI.

**Comment by winterheart** [comment]
@mgorny fixed

**Comment by juippis** [comment]
Good work, thanks!

**Pull request closed without merging**